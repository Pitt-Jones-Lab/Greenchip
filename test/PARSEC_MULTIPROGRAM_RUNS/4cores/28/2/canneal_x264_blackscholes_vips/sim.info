{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/2/canneal_x264_blackscholes_vips',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_2_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/2/canneal_x264_blackscholes_vips',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpgpYruN/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb17',
 'pin_version': 'unknown',
 'rusage': (17640.058303000002,
            1019.8139640000001,
            333808,
            0,
            0,
            0,
            80587,
            2,
            0,
            10744,
            1376,
            0,
            0,
            0,
            126692153,
            5696464),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/2/canneal_x264_blackscholes_vips --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem28.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_2_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpgpYruN/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/2/canneal_x264_blackscholes_vips/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 8459.8900120258331,
 't_start': 1463065526.286206,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1746776064L}
