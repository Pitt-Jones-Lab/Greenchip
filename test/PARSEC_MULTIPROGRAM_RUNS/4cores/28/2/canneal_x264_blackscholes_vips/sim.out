                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  1703213505 |  2546205917 |  2815753961 |  2933467214
  Cycles                           |  1662129994 |  1662129994 |  1662129994 |  1662129994
  IPC                              |        1.02 |        1.53 |        1.69 |        1.76
  Time (ns)                        |   624860900 |   624860900 |   624860900 |   624860900
  Idle time (ns)                   |      101333 |       55304 |      266238 |      836928
  Idle time (%)                    |        0.0% |        0.0% |        0.0% |        0.1%
Branch predictor stats             |             |             |             |            
  num correct                      |   167076698 |   135633355 |   334424337 |   247084655
  num incorrect                    |      198705 |     7757247 |     9898698 |     2201493
  misprediction rate               |       0.12% |       5.41% |       2.87% |       0.88%
  mpki                             |        0.12 |        3.05 |        3.52 |        0.75
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   221311597 |   287930801 |   386586513 |   356069583
    num misses                     |        1964 |       13643 |        5350 |       12605
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.01 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   551001961 |   832854769 |   814242260 |   846858321
    num misses                     |      193874 |     3966245 |      209295 |     3618713
    miss rate                      |       0.04% |       0.48% |       0.03% |       0.43%
    mpki                           |        0.11 |        1.56 |        0.07 |        1.23
  L2 TLB                           |             |             |             |            
    num accesses                   |      195838 |     3979888 |      214645 |     3631318
    num misses                     |      126801 |      646087 |       86826 |      593113
    miss rate                      |      64.75% |      16.23% |      40.45% |      16.33%
    mpki                           |        0.07 |        0.25 |        0.03 |        0.20
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   221311597 |   287930801 |   386586513 |   356069583
    num cache misses               |      114848 |    15061706 |      253369 |     7224330
    miss rate                      |       0.05% |       5.23% |       0.07% |       2.03%
    mpki                           |        0.07 |        5.92 |        0.09 |        2.46
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   551001961 |   832854769 |   814242260 |   846858321
    num cache misses               |     8562124 |    16955133 |     7195822 |    15676208
    miss rate                      |       1.55% |       2.04% |       0.88% |       1.85%
    mpki                           |        5.03 |        6.66 |        2.56 |        5.34
  Cache L2                         |             |             |             |            
    num cache accesses             |    12601937 |    32875557 |     9031575 |    24787222
    num cache misses               |     6325241 |     5842270 |     3706929 |     5640267
    miss rate                      |      50.19% |      17.77% |      41.04% |      22.75%
    mpki                           |        3.71 |        2.29 |        1.32 |        1.92
DRAM summary                       |             |             |             |            
  num dram accesses                |    37850824 |           0 |           0 |           0
  average dram access latency (ns) |      114.18 |         inf |         inf |         inf
  average dram queueing delay      |      114.18 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     5157943 |     4316618 |     3358581 |     5115863
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
